src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -241,17 +247,31 @@ To avoid these issues, set wait conditions that will pause the script until the
   >
     Instruct the monitor to wait for a page element to be present. This is useful when an element your script depends on takes a long time to load, or simply to ensure that a page element is present before your script clicks on it.
 
-    Use any of the [locate functions](#locating-elements) with the wait function below. Use your desired function instead of the `$driver.By.linkText` function used below:
+    Use any of the [locate functions](#locating-elements) with the wait function below. Use your desired function instead of the `$selenium.By.linkText` function used below:
 
     ```
-    //Tell the monitor to wait up to 7,500 milliseconds (7.5 seconds) for a link whose text is `Your Link Text Here`.
-    $browser.waitForAndFindElement($driver.By.linkText("Your Link Text Here"), 7500);
+    // Tell the monitor to wait up to 7,500 milliseconds (7.5 seconds) for a link whose text is `Your Link Text Here`.
+    var linkText = "Your Link Text Here"
+
+    var element = await $webDriver.wait(
+      $selenium.until.elementLocated(
+        $selenium.By.linkText(linkText),
+      ),
+      7500,
+      'Timed-out waiting for element to be located using link text: "' +linkText+ '"',
+    );
+    // Wait for the element to be visible
+    await $webDriver.wait(
+      $selenium.until.elementIsVisible(element),
+      7500,
+      "Timed-out waiting for element to be visible using ${element}",
+    );
     ```
 
     For an example of waiting for a specific page element to load, see [Wait for a page element](/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors/#wait-page-element).
```

Comment:
There are a couple of lines on this page which seem to link back to their own section, or no specific section on the same page. Should these be removed?

Reply:
Yes, I've removed both.